js/ui/grid_core/ui.grid_core.validating.js

```diff
@@ -798,6 +798,22 @@ module.exports = {
                     }
                 },
 
+                highlightDataCell: function($cell, parameters) {
+                    const isEditableCell = parameters.setValue;
+                    const cellModified = this.isCellModified(parameters);
+
+                    if(!cellModified && isEditableCell) {
+                        const result = this.getController('validating').getCellValidationResult({
+                            rowKey: parameters.key,
+                            columnIndex: parameters.column.index
+                        });
+                        const skipValidation = parameters.row.isNewRow || !isDefined(result);
```

Comment:
Extract `isDefined(result)` to variable `isValidated`

Reply:
OK